global/terminal/double_click_windows.go

```diff
@@ -59,13 +61,34 @@ func NoMoreDoubleClick() error {
 
 // BoxW of Win32 API. Check https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-messageboxw for more detail.
 func boxW(hwnd uintptr, caption, title string, flags uint) int {
-	captionPtr, _ := syscall.UTF16PtrFromString(caption)
-	titlePtr, _ := syscall.UTF16PtrFromString(title)
-	ret, _, _ := syscall.NewLazyDLL("user32.dll").NewProc("MessageBoxW").Call(
+	captionPtr, _ := windows.UTF16PtrFromString(caption)
+	titlePtr, _ := windows.UTF16PtrFromString(title)
+	u32 := windows.NewLazySystemDLL("user32.dll")
```

Comment:
add comment about why not using package syscall

Reply:
As written in the [documentation](https://pkg.go.dev/syscall?GOOS=windows), 

>Deprecated: this package is locked down. Callers should use the corresponding package in the golang.org/x/sys repository instead. That is also where updates required by new systems or versions should be applied. See https://golang.org/s/go1.4-syscall for more information.

package `syscall` is deprecated and package `golang.org/x/sys`  is the recommended alternative.